pkgs/development/python-modules/nevow/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k, twisted }:
+
+buildPythonPackage rec {
+  name = "nevow-${version}";
```

Comment:
Drop this line.

Reply:
I dropped that before  but then the package name is capitalized (it's capitalized on pypi). If we do that, it will be called python2.7-Nevow. Do we want that?

Reply:
Yes we do. It isn't relevant anyway. The advantage of not having `name` is that we can easily override the version.